### ISSUE TITLE
Need to track tasks on app start

### ISSUE THREAD

**Issue by svijaykoushik** [svijaykoushik/chrono-chime-desktop#30]
### Track tasks on app start

Tasks that have been started, begin to start tracking only after visiting tasks page. this should happen on app start.

**Comment by svijaykoushik** [comment]
Closing as out of scope.

ChronoChime's PRD explicitly names **Task lists** under [Section 10 — Out of Scope](docs/prd/chronochime-product-requirements-document.md), and AGENTS.md treats task management as a non-negotiable guardrail. The product answers two questions only: *when* should the user be notified, and *how*. Task management is a separate product.

The v1 rewrite removed task management entirely — there are no task/todo code paths in `src/`.
